src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -1,12 +1,12 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Machine;
 import java.util.List;
-
 /**
  * Parameterize this service and add its implementation in a separate class.
  */
-public interface MachineService {
 
+public interface MachineService<T extends Machine> {
```

Comment:
Don't begin class or method implementation with an empty line. Remove the empty lines (10-17) before the first method declaration.